tools/compute_image_mean.cpp

```diff
@@ -115,5 +116,8 @@ int main(int argc, char** argv) {
     }
     LOG(INFO) << "mean_value channel [" << c << "]:" << mean_values[c] / dim;
   }
+#else
+  LOG(FATAL) << "Compile with OpenCV to use this.";
```

Comment:
"This tool requires OpenCV." would match the text for the examples although it might be best if the message everywhere were "This [whatever] requires [dependency]: compile with [dependency] to use."